.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: ''
+labels: bug, awaiting triage
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem else remove this section.
+
+**Desktop (please complete the following information else remove this section.):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information else remove this section.):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
```

Comment:
We shouldn't ask them to just say "stock browser". Let them tell what browser they used specifically.

```suggestion
 - Browser [e.g. safari, firefox, chrome]
```